src/actions.js

```diff
@@ -300,7 +341,10 @@ export function fetchClaim(mm, claimUuid, forFeedback) {
   } else {
     projections.push(
       "services{" +
-        "id, product { id, uuid }, service {id code name price maximumAmount} qtyProvided, priceAsked, qtyApproved, priceApproved, priceValuated, priceAdjusted, explanation, justification, rejectionReason, status" +
+        "id, product { id, uuid }, service {id code name price maximumAmount packagetype} qtyProvided,  priceAsked, qtyApproved, priceApproved, priceValuated,priceAdjusted, explanation, justification, rejectionReason, status," +
+        " claimlinkedItem{ item { id code name } qtyDisplayed priceAsked qtyProvided }"+
```

Comment:
I would rewrite it as a separate projection to be concatenated, @olewandowski1 what do you think?